gravitee-apim-console-webui/src/portal/navigation/portal-navigation.service.ts

```diff
@@ -60,6 +60,13 @@ export class PortalNavigationService {
         icon: 'gio:report-columns',
       });
     }
+    if (this.permissionService.hasAnyMatching(['environment-settings-r', 'environment-settings-u'])) {
+      customization.children.push({
+        displayName: 'API',
+        routerLink: 'api',
+        icon: 'gio:cloud-settings',
+      });
+    }
```

Comment:
If you'd like, the API customization uses the same settings as the Top Bar customization. So they could be combined in the same `if`

Reply:
Done